oci/table_oci_core_image.go

```diff
@@ -130,13 +130,15 @@ func tableCoreImage(_ context.Context) *plugin.Table {
 				Name:        "compartment_id",
 				Description: ColumnDescriptionCompartment,
 				Type:        proto.ColumnType_STRING,
-				Transform:   transform.FromField("CompartmentId"),
+				Hydrate:     getTenantId,
+				Transform:   transform.FromValue(),
 			},
 			{
 				Name:        "tenant_id",
 				Description: ColumnDescriptionTenant,
 				Type:        proto.ColumnType_STRING,
-				Transform:   transform.FromField("CompartmentId"),
+				Hydrate:     getTenantId,
+				Transform:   transform.FromValue(),
```

Comment:
Value will be same for both the columns, then do we need both the columns?

Reply:
as discussed we will keep both for now.